CHANGELOG.md

```diff
@@ -3,6 +3,20 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
-## Unreleased
+## Development Release 0.15.0.dev11 [2018-05-25]
```

Comment:
All of it should be under "Unreleased". I don't want to describe each individual development snapshot.